rockspec/resty-redis-cluster-1.02-4.rockspec

```diff
@@ -1,21 +1,18 @@
 package = "resty-redis-cluster"
-version = "1.0-1"
+version = "1.02-4"
 source = {
-    url = "https://github.com/steve0511/resty-redis-cluster/"
+    url = "git://github.com/steve0511/resty-redis-cluster",
+    tag = "1.02",
 }
 description = {
     summary = "Openresty lua client for redis cluster",
     detailed = [[
         Openresty environment lua client with redis cluster support.
-        This is a wrapper around the 'resty.redis' library with cluster discovery 
+        This is a wrapper around the 'resty.redis' library with cluster discovery
         and failover recovery support.
     ]],
     license = "Apache License 2.0"
 }
-dependencies = {
```

Comment:
I remove this field `dependencies`, it is wrong if the user ENV is openresty.
we can write a doc if the user uses other ENV.